skbio/alignment/__init__.py

```diff
@@ -218,11 +220,14 @@
     local_pairwise_align_ssw,
 )
 from skbio.alignment._ssw_wrapper import StripedSmithWaterman, AlignmentStructure
+from skbio.alignment._path import AlignPath, PairAlignPath
 
 __all__ = [
     "TabularMSA",
     "StripedSmithWaterman",
     "AlignmentStructure",
```

Comment:
Just realized that there is an `AlignmentStructure` class. Outdated. Should be removed when we remove the SSW wrapper code. The new `AlignPath` class can replace it.